src/hybrid_reader.h

```diff
@@ -37,7 +40,7 @@ typedef struct {
 extern "C" {
 #endif
 
-IndexIterator *NewHybridVectorIterator(VecSimIndex *index, char *score_field, KNNVectorQuery query, VecSimQueryParams qParams, IndexIterator *child_it);
+IndexIterator *NewHybridVectorIterator(VecSimIndex *index, char *score_field, KNNVectorQuery query, size_t dim, VecSimType type, VecSimMetric metric, VecSimQueryParams qParams, IndexIterator *child_it);
```

Comment:
can you pass the &info instead of a big list of parameters?

Reply:
I can, but then we will need a switch inside `NewHybridVectorIterator` again and we only need this three values